hubblestack/extmods/modules/hubble.py

```diff
@@ -209,7 +209,7 @@ def _build_terse_results(ret, show_success, show_compliance):
         tag = tag_data['tag']
         description = tag_data.get('description')
         if (tag, description) not in tags_descriptions:
-            terse_results['Failure'].append({tag: description})
+            terse_results['Failure'].append({tag: description, 'detail': tag_data['detail']})
```

Comment:
Can we make this a little more generic to have this support for all other modules as well. The **detail** key will now be added to all modules here, but in returner, it will only cater to oval_scanner's params since you are checking for very specific keys over there.

Reply:
@MoodyMudit what generic term should we use?  I'm not that picky about it, but it might be a little subjective so I want to make sure we use something everyone can agree on.  I take it we just want to rename the **detail** key to something else, correct?

Reply:
No. Please read me comment here https://github.com/hubblestack/hubble/pull/910#issuecomment-699533246. the keyword **details** is fine, but in the returner, you are only adding these details when these details are sent from the oval_scanner. Can we not loop and add all available key values so that I can have different details from different modules. I think you got confused because I commented on this part of the diff. I should have made this comment in **splunk _nova_return**